### ISSUE TITLE
Remove the warning

### ISSUE THREAD

**Issue by curquiza** [meilisearch/meilisearch-rust#17]
The warning should be removed

```bash
$ cargo build
warning: unused import: `settings::*`
 --> src/indexes.rs:2:85
  |
2 |     client::Client, document::*, errors::Error, progress::*, request::*, search::*, settings::*,
  |                                                                                     ^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

    Finished dev [unoptimized + debuginfo] target(s) in 0.08s
```

This warning also appears during the tests launch
